### DISCUSSION TITLE
ΛCDM is alive and well

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#242] [Cosmology Models]
"ΛCDM is alive and well" arXiv:2205.05017
https://arxiv.org/abs/2205.05017

They say that so nobody will panic - this is common practice in dire situations.
Cosmologists still hope there will be enough lifeboats...

P.S. Sorry, I can't resist updating the meme.
![LCDM-Alive-and-Well-z14 3_Titanic](https://github.com/a-cosmology-group/acg/assets/16748402/8624345d-6d7b-49e2-b11d-12dd22d1372e)

**Comment by budrap00** [comment]
Just like Elvis!

**Comment by LifeIsStrange** [comment]
The paper seems to refute or at least show new tensions with most lcdm extensions.
Lcdm is clearly dead via many independent probes and it's only hope are its extensions so it doesn't bode well.
The title also reminds me of
https://arxiv.org/abs/2310.03063

Conversely here's a paper with a play on word saying to put LCDM in the bin (trashcan)
https://arxiv.org/abs/2206.11447
Because it is no longer a model with constant integration constants.

**Reply by RedshiftDrift** [reply]
The Big Bang is the zombie that never dies [www.nature.com/articles/377099a0](https://www.nature.com/articles/377099a0)...

**Comment by HanDeBruijn** [comment]
From the picture: with [z=28.5](http://www.alternatievewiskunde.nl/cosmology/helaas.htm#rood) the Big Boat has sunk completely.
